scripts/upgrade

```diff
@@ -12,15 +12,28 @@ domain=$(ynh_app_setting_get "$app" domain)
 path=$(ynh_app_setting_get "$app" path)
 is_public=$(ynh_app_setting_get "$app" is_public)
 
+### This helper will compare the version of the currently installed app and the version of the upstream package.
+### $upgrade_type can have 2 different values
+### - UPGRADE_APP if the upstream app version has changed
+### - UPGRADE_PACKAGE if only the YunoHost package has changed
+### ynh_check_app_version_changed will stop the upgrade if the app is up to date.
+### UPGRADE_APP should be used to upgrade the core app only if there's an upgrade to do.
+upgrade_type=$(ynh_check_app_version_changed)
+
 # Remove trailing "/" for next commands
 path=${path%/}
 
 # Retreive sources and install them
 src_path=/var/www/$app
 sudo rm -rf $src_path
 sudo mkdir -p $src_path
-url=$(curl -s https://api.github.com/repos/duniter/cesium/releases | grep "browser_" | grep "web" | head -1 | cut -d\" -f4)
-wget -nc --quiet $url -P /tmp
+if [ "$upgrade_type" == "UPGRADE_APP" ]
+then
+	ynh_script_progression --message="Upgrading source files..." --time --weight=1
+
+	# Download, check integrity, uncompress and patch the source from app.src
+	ynh_setup_source --dest_dir=/tmp
```

Comment:
This helper will uncompress the downloaded file, so you don't have to do it by hand with:

```
sudo unzip -q /tmp/cesium-*-web.zip -d $src_path
sudo rm -f /tmp/cesium-*-web.zip
```


You should replace this by:
```suggestion
	ynh_setup_source --dest_dir=$src_path
```

Btw, same work can be done on the install script

Reply:
Thank you, I have made the changes you said.
And I'll look into install script tomorrow.